skulpt.py

```diff
@@ -240,6 +240,29 @@ def test(debug_mode=False):
         ret4 = rununits(debug_mode=debug_mode)
     return ret1 | ret2 | ret3 | ret4
 
+def test3(debug_mode=False):
```

Comment:
Yes.  It was unclear if they would diverge further in the future, but I guess there is a lot of repeated code this way.